trunk/SUAVE/Analyses/Weights/Weights.py

```diff
@@ -24,15 +25,20 @@ def __defaults__(self):
         self.vehicle  = Data()
         
         self.settings = Data()
-        self.settings.empty_weight_method = \
-            SUAVE.Methods.Weights.Correlations.Tube_Wing.empty
         
         
     def evaluate(self,conditions=None):
         
         # unpack
         vehicle = self.vehicle
-        empty   = self.settings.empty_weight_method
+        
+        if vehicle.fuselages.keys() == []:

```

Comment:
I think we should be settings these with separate analysis classes rather than relying on the fuselage name.